### ISSUE TITLE
workers

### ISSUE THREAD

**Issue by seungbo** [ultralytics/ultralytics#17686]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

<img width="1083" alt="스크린샷 2024-11-21 오후 5 04 59" src="https://github.com/user-attachments/assets/b3fcd228-3b57-4142-b8ce-3c74bc6a361e">

<img width="222" alt="스크린샷 2024-11-21 오후 5 10 06" src="https://github.com/user-attachments/assets/a99fb7ef-1f14-474d-a377-96ec0ea06d38">

Why am I getting the warning shown in the photo?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @seungbo, thank you for reaching out to us about your question on the Ultralytics repository 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) where you might find helpful explanations and examples for common warnings and errors.

If this is a 🐛 Bug Report causing the warning, could you please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) that triggers this warning? This will help us in diagnosing the issue more efficiently. 

For custom training ❓ queries, having detailed context, such as dataset samples, configuration, and logs, can be invaluable. Ensure you're familiar with our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Feel free to connect with the Ultralytics community for more insights and support. Join us on [Discord](https://ultralytics.com/discord) 🎧 for real-time communications. Engage with detailed discussions on [Discourse](https://community.ultralytics.com), or delve into our [Subreddit](https://reddit.com/r/ultralytics) to exchange ideas!

## Upgrade

Make sure you're using the latest version of the `ultralytics` package by upgrading with:

```bash
pip install -U ultralytics
```

Verify all installation requirements from [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) are met in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/).

## Environments

YOLO can be run across numerous versatile environments that are preloaded with all essential dependencies:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Refer to [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Refer to [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. Follow the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

A green status on this badge indicates all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. These tests run every 24 hours and with each commit to verify the correct operation across platforms.

Please note, this is an automated response. An Ultralytics engineer will assist you soon. We appreciate your patience and interest in our project! 🚀

**Comment by Y-T-G** [comment]
You can set it to `workers=2`.

**Comment by seungbo** [comment]
However, I am using a GPU H100.

**Comment by Y-T-G** [comment]
Can you provide the output after running this command in terminal: `yolo checks`?

**Comment by seungbo** [comment]
<img width="719" alt="스크린샷 2024-11-25 오전 8 10 10" src="https://github.com/user-attachments/assets/b17e0502-35e8-45c6-a363-908288838f4a">

**Comment by Y-T-G** [comment]
You can ignore the warning then. It's not from `ultralytics`. It's from `torch`
